src/main/java/com/deavensoft/paketomat/paketomats/PaketomatService.java

```diff
@@ -0,0 +1,15 @@
+package com.deavensoft.paketomat.paketomats;
+
+
+import com.deavensoft.paketomat.center.model.Paketomat;
+
+import java.util.List;
+
+public interface PaketomatService {
+
+    List<Paketomat> getAllPaketomats();
+
+    void savePaketomats(Paketomat paketomat);
```

Comment:
According to the method name, I expect that I'm able to save a list of paketomats, but I notice Paketomat as a parameter of method. 
Change the method name or the parameter type.

Reply:
Changed name